src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -66,6 +68,7 @@
 			public static final double TURN_SPEED_SCALE = 1;
 			public static final double XBOX_MINIMUM_THRESHOLD = 0.1;
 			public static final double OPERATOR_JOYSTICK_MINIMUM_THRESHOLD = 0.1;
+			public static final double CLIMBER_TARGET_SPEED = 0.65;
```

Comment:
@HowardStark et al. will throw a fit if you keep this constant here. Consider moving it to the `Climber` class?
```java
public class Climber {
    public static final double TARGET_SPEED = 0.65;
}
```